### PR TITLE
Add `isConnected` and `getRootNode()` to Node

### PR DESCRIPTION
**Pull request by jorgecasar**
Solve #7180

### CODE REVIEW

File: lib/dom.js [hunk 1]
```diff
@@ -500,6 +501,7 @@ declare class Node extends EventTarget {
   cloneNode(deep?: boolean): this;
   compareDocumentPosition(other: Node): number;
   contains(other: ?Node): boolean;
+  getRootNode(): Node;
```
Comment (jbrown215) [inline lib/dom.js]: https://developer.mozilla.org/en-US/docs/Web/API/Node/getRootNode

Can you add the optional object argument?

Comment (jbrown215) [reply lib/dom.js]: `void | {composed: boolean}` is what you want, since it's either omitted or or an object with a `composed` field.

Comment (jorgecasar) [reply lib/dom.js]: Thanks, I'm not familiarized with flow, sorry.

Just to be sure, this: `getRootNode(options: void | {composed: boolean}): Node;` is the same than `getRootNode(options?: {composed: boolean}): Node;`

Comment (jbrown215) [reply lib/dom.js]: No problem! Best way to learn is to write libdefs like this.

Those two types are equivalent to callers, yes. I'm not sure how our function subtyping handles optional arguments vs. void union arguments, but `getRootNode(options?: {composed: boolean}): Node;` is totally fine for what we need here.

**Comment by jbrown215** [comment]
The tests need to be re-recorded, since error messages have been updated.
I can handle that. Thanks for this again!

**Comment by jorgecasar** [comment]
Thanks!

**Pull request closed without merging**